eventlogger/src/main/java/gov/cms/ab2d/eventlogger/events/ErrorEvent.java

```diff
@@ -1,6 +1,6 @@
 package gov.cms.ab2d.eventlogger.events;
 
-import gov.cms.ab2d.eventlogger.LoggableEvent;
```

Comment:
Why isn't this one deleted? Is it not in the event client?

Reply:
Made changes to event libs to resolve this. https://github.com/CMSgov/AB2D-Libs/pull/105